README.md

```diff
@@ -50,6 +50,53 @@ If you have Vagrant installed in your machine, provision and run a virtual machi
 $ vagrant up
 ```
 This will automatically launch the app, which you can also view in your [`browser`](http://localhost:5000/).
+
+## Running with Docker
+### Dev
+In developer mode, Flask tracks changes to the files in the codebase. You should see the changes after a simple tab refresh, without the need to rebuild the docker image.
+
+To run in developer mode, first build the docker image with the following command from the project directory:
+```bash
+docker build --target development --tag todo-app:dev .
+```
+After you've built the image, run the website with the following command:
+```bash
+docker run -p 5000:5000 --env-file .env --mount type=bind,source="$(pwd)",destination=/todo-app todo-app:dev
```

Comment:
You don't need `--env-file .env` here because the whole directory is mounted. .dockerignore doesn't affect the bind mount.